constructor/imaging.py

```diff
@@ -5,12 +5,15 @@
 # Consult LICENSE.txt or http://opensource.org/licenses/BSD-3-Clause.
 
 import sys
+from io import BytesIO
 from os.path import dirname, join
 from random import randint
 
 from PIL import Image, ImageDraw, ImageFont
 
 ttf_path = join(dirname(__file__), 'ttf', 'Vera.ttf')
+with open(ttf_path, "rb") as f:
+    ttf_bytes = f.read()
```

Comment:
Would be worthwhile to use https://docs.python.org/3/library/importlib.resources.html for this?

Reply:
One day, yes. That path has worked for a long time so I just want to unblock this issue to release 3.9.1.